draft-ietf-rtcweb-jsep.xml

```diff
@@ -225,7 +225,7 @@
     <section title="Semantics and Syntax" anchor="sec.semantics-and-syntax">
       <section title="Signaling Model" anchor="sec.signaling-model">
         <t>JSEP does not specify a particular signaling model or state
-        machine, other than the generic need to exchange SDP media
+        machine, other than the generic need to exchange SDP session
```

Comment:
SDP and "session description" seem redundant. I would remove "SDP" and just say "session description" here and below.